kubeadm-one

```diff
@@ -305,14 +314,27 @@ function initialize_kubeadm() {
 
 function configure_kubernetes() {
 
+    pod_network_cidr="$1"
+    container_network_interface="$2"
+
     # Configure kubeconfig for remainder of script
     export KUBECONFIG=/etc/kubernetes/admin.conf
 
     ## Remove master taints
     kubectl taint nodes --all node-role.kubernetes.io/master- || true
 
-    ## Deploy a pod network (e.g. flannel)
-    kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml
+    container_network_interface="calico"
+    ## Deploy a pod network
+    if [[ "$container_network_interface" == "calico" ]]; then
```

Comment:
In the future, a `case` statement might be better here.  I'm cool with how it is for now.